app/services/facets/entry.rb

```diff
@@ -0,0 +1,67 @@
+module Facets
+  class Entry
+    attr_accessor :name, :model, :helper, :extension,
```

Comment:
I would tend to agree as even these checks aren't completely fool proof.  I could pass a model class that isn't actually active record, same for the controller check.  You could check the base class but i feel that is heading down the a rabbit whole. 

I would prefer to see the facet API well documented here with good usage examples.  I think that is more valuable than validation checks.